ports/mimxrt/led.c

```diff
@@ -0,0 +1,193 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2020 Philipp Ebensberger
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+#include "fsl_gpio.h"
+#include "fsl_iomuxc.h"
+
+#include "py/runtime.h"
+#include "py/mphal.h"
+#include "led.h"
+#include "pin.h"
+
+/// \moduleref machine
+/// \class LED - LED object
+///
+/// The LED object controls an individual LED (Light Emitting Diode).
```

Comment:
These triple-slash quotes are left over from long ago when the docs were automatically extracted from the C code, and are no longer needed, nor are the meta tags like `\moduleref` and `\class`.  I understand this was copied from code in stm32 and that should be removed there.  But for now please just use normal `//` comments and none of the meta data.

Reply:
Yes, I took stm32 as a reference. Will remove these kind of comments 👌